### DISCUSSION TITLE
[Feature request] Pie chart top

### DISCUSSION THREAD

**Discussion by Tepew** [grafana/grafana#41875] [Dashboards, Panels & Visualizations]
Now we have a lot of requests for a demo of the top 10 or 15. With the datasource from the database, we made such a solution, but it is cumbersome and requires corrections every time. If this is a functional, it will be a great solution.
And add check box for sort other in end of table f.e.:
1 - 100
2 - 40
3 - 30 
4 - 15
5 - 5
other - 30
Top size - add user
name of other - defaukt, but user can change

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
